### PR TITLE
Add support for CV failure notification type

### PR DESCRIPTION
**Pull request by rexredinger**
## Testing plan

N/A

## External links

- [API documentation](https://github.com/hashicorp/terraform-docs-common/pull/175)

## Output from tests
Including output from tests may require access to a TFE instance. Ignore this section if you have no environment to test against.

```
$ TFE_ADDRESS="https://example" TFE_TOKEN="example" TF_ACC="1" go test ./... -v -tags=integration -run TestFunctionsAffectedByChange

...
```

### CODE REVIEW

**Review by sebasslash — COMMENTED** [review]
Can you add a changelog entry? Currently there are no unreleased changes (it should say `v1.11.0`), so it would be an awesome favor if you could update that as well :heart:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
-# (Unreleased)
+# v1.11.0
```
Comment (rexredinger) [inline CHANGELOG.md:1]: @sebasslash  Excuse me if I misinterpreted, but this is OK?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 
 ## Enhancements
 
 * Add `Query` and `Status` fields to `OrganizationMembershipListOptions` to allow filtering memberships by status or username by @sebasslash [#550](https://github.com/hashicorp/go-tfe/pull/550)
 * Add `ListForWorkspace` method to `VariableSets` interface to enable fetching variable sets associated with a workspace by @tstapler [#552](https://github.com/hashicorp/go-tfe/pull/552)
 * Add `NotificationTriggerAssessmentDrifted` and `NotificationTriggerAssessmentFailed` notification trigger types by @lawliet89 [#542](https://github.com/hashicorp/go-tfe/pull/542)
+* Add ``NotificationTriggerAssessmentCheckFailed` trigger type by @rexredinger [#549](https://github.com/hashicorp/go-tfe/pull/549)
```
[the remark this answers is not in the document]
Comment (sebasslash) [reply CHANGELOG.md:8]: Everything currently under the "Unreleased" section was released in v1.11.0

**Review by Uk1288 — APPROVED** [review]

**Review by sebasslash — APPROVED** [review]
:rocket:

**Pull request merged**